cluster-scope/base/rbac.authorization.k8s.io/clusterroles/service-catalog-prometheus-plugin/clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: service-catalog-prometheus-plugin
```

Comment:
Why do we need a new cluster role here? Isn't `cluster-monitoring-view` enough?

Reply:
Didn't notice that it exists thanks :+1:. Should be changed now.